packages/react-devtools-core/src/standalone.js

```diff
@@ -44,6 +41,12 @@ function reload() {
   ReactDOM.unmountComponentAtNode(node);
   node.innerHTML = '';
   setTimeout(() => {
+    if (typeof window.__REACT_DEVTOOLS_GLOBAL_HOOK__ === 'undefined') {
+      window.__REACT_DEVTOOLS_GLOBAL_HOOK__ = {};
```

Comment:
We should never need to do this.
`installGlobalHook` is guaranteed to set it (which is called at the beginning).